samples/tfm_integration/psa_level_1/README.rst

```diff
@@ -0,0 +1,230 @@
+.. _tfm_psa_level_1:
+
+TF-M PSA Level 1
+################
+
+Overview
+********
+This TF-M integration example can be used with an Arm v8-M supported board. It
+demonstrates how to use certain TF-M features that are covered as part of the
+RTOS vendor requirements for a `PSA Certified Level 1`_ product, such as
+secure storage for config data, initial attestation for device verification,
+and the PSA crypto API for cryptography.
+
+Trusted Firmware (TF-M) Platform Security Architecture (PSA) APIs
+are used for the secure processing environment, with Zephyr running in the
+non-secure processing environment.
+
+As part of the standard build process, the secure bootloader (BL2) is built, as
+well as the secure TF-M binary and non-secure Zephyr binary images. The two
+application images are then merged and signed using the public/private key pair
+stored in the secure bootloader, so that they will be accepted during the
+image validation process at startup.
+
+All TF-M dependencies will be automatically cloned in the ``/ext`` folder, and
+built if appropriate build artefacts are not detected.
+
+.. _PSA Certified Level 1:
+  https://www.psacertified.org/security-certification/psa-certified-level-1/
+
+.. note:: This example is based upon trusted-firmware-m 1.0-rc2 (commit 4117e0353c55dce739f550515c5b007411142ba7)
+
+Building and Running
+********************
+
+This project outputs startup status and info to the console. It can be built and
+executed on an MPS2+ configured for AN521 (dual-core ARM Cortex M33).
+
+This sample will only build on a Linux or macOS development system
+(not Windows), and has been tested on the following setups:
+
+- macOS Mojave using QEMU 4.1.0 with gcc-arm-none-eabi-7-2018-q2-update
+- macOS Mojave with gcc-arm-none-eabi-7-2018-q2-update
+- Ubuntu 18.04 using Zephyr SDK 0.10.1
+
+On MPS2+ AN521:
+===============
+
+1. Build Zephyr with a non-secure configuration
```

Comment:
The "using west" and "using cmake" instructions are part of step 1, so should be indented:

```
1. Build Zephyr with a non-secure configuration
   (``-DBOARD=mps2_an521_nonsecure``). This will also build TF-M in the
   ``ext/tfm`` folder if necessary.

   Using ``west``:

   .. code-block:: bash

      cd <ZEPHYR_ROOT>/samples/tfm_integration/psa_level_1/
      west build -p -b mps2_an521_nonsecure ./

   Using ``cmake``:

   .. code-block:: bash

      cd <ZEPHYR_ROOT>/samples/tfm_integration/psa_level_1/
      mkdir build
      cd build
      cmake -GNinja -DBOARD=mps2_an521_nonsecure ..
      ninja -v
```